src/vs/base/common/async.ts

```diff
@@ -238,6 +238,15 @@ export class Delayer<T> {
 		}
 	}
 
+	fire(): void {
```

Comment:
I think this isn't used anymore right?

Reply:
Yes, removed the blur event emitter but wasn't sure whether to remove this as it might be useful to keep so waited for comments. Better to remove altogether if not used?

Reply:
Yes, removed the blur event emitter but wasn't sure whether to remove this as it might be useful to keep so waited for comments. Will remove based on your other comment.